src/Parser.jl

```diff
@@ -262,6 +262,7 @@ function read_unicode_escape!(ps)
     end
 end
 
+get_bytes(str) = Vector{UInt8}(@static isdefined(Base, :codeunits) ? codeunits(str) : str)
```

Comment:
This function is not necessary given the changes I suggest below.

However, it would be good to have a `Compat.codeunits(s)` function that just returns `Vector{UInt8}(s)` in Julia 0.6.

Reply:
Just pushed a PR for Compat to add `codeunits` and `ncodeunits`

Reply:
If those are merged soon, then yes, I'll change this to use codeunits.